processing/src/main/java/io/druid/query/groupby/epinephelinae/ConcurrentGrouper.java

```diff
@@ -94,6 +96,8 @@ public ConcurrentGrouper(
           )
       );
     }
+
+    this.entryComparator = keySerdeFactory.factorize().entryComparator();
```

Comment:
@gianm updated with renames 
comparator() to bufferComparator()
entryComparator() to objectComparator() and moved it to factory , made it comparable of T rather than Entry<T> .